mitmproxy/proxy/layers/http/_http2.py

```diff
@@ -275,6 +275,8 @@ def handle_h2_event(self, event: h2.events.Event) -> CommandGenerator[bool]:
             # https://http2.github.io/http2-spec/#rfc.section.4.1
             # Implementations MUST ignore and discard any frame that has a type that is unknown.
             yield Log(f"Ignoring unknown HTTP/2 frame type: {event.frame.type}")
+        elif isinstance(event, h2.events.AlternativeServiceAvailable):
+            yield Log("Temporarily ignoring an Alt-Svc header")
```

Comment:
Let's reduce this to DEBUG verbosity (most people will not care) and make the message a bit clearer.
```suggestion
            yield Log("Received HTTP/2 Alt-Svc frame, which will not be forwarded.", DEBUG)
```